pkg/liqonet/liqonet_suite_test.go

```diff
@@ -0,0 +1,38 @@
+package liqonet_test
```

Comment:
Do we really need the controller-runtime environment to perform this simple test? I think a [FakeClientSet](https://pkg.go.dev/k8s.io/client-go/kubernetes/fake#NewSimpleClientset) would fit better without the necessity to set up all this environment

Reply:
I took into consideration the fake client, but as far as I understood you need to declare the objects at the beginning, so the third test is not possible. Do you agree or am I missing something?

Reply:
You can instantiate a client and then create resources wherever you want, there are no constraints.